plugins/feeds/public/abusech_malwarebazaar.py

```diff
@@ -63,14 +63,22 @@ def analyze(self, block):  # pylint: disable=arguments-differ
         context["imphash"] = block["hash"]["imphash"]
         context["ssdeep"] = block["hash"]["ssdeep"]
 
-        if block.get("hash", []):
+        if block.get("sha256_hash", ""):
+            malware_file = File.get_or_create(
+                value='FILE:{}'.format(block["sha256_hash"]))
+
+            malware_file.add_context(context)
+            malware_file.tag(block["file_type_mime"])
             # md5, sha1, sha256
-            for hash_type in ("md5_hash", "sha1_hash", "sha256_hash"):
+            for hash_type in ("md5_hash", "sha1_hash", "sha256_hash") or []:
```

Comment:
Why the ` or  []`? You're manually specifying the list so it will never evaluate to False

Reply:
ah true lol, watchign tv and coding is bad